docs/KNOWN_BUGS

```diff
@@ -527,6 +529,16 @@ problems may have been fixed or changed somewhat since this was written.
 
  See https://github.com/curl/curl/issues/13350
 
+11.7 macidn test failures
+
+ Test 1034 and 1035 fail on macOS when built to use macidn.
+
+ See https://github.com/curl/curl/issues/14176
+
+11.8 winidn test failures
```

Comment:
```suggestion
11.7 AppleIDN test failures

 Test 1034 and 1035 fail on macOS when built to use macidn.

 See https://github.com/curl/curl/issues/14176

11.8 WinIDN test failures
```
for consistency. It could be MacIDN, too, but it applies to all Apple platforms, not just Macs.